client/src/remote/Backend.js

```diff
@@ -8,8 +8,9 @@ const ids = new Ids();
  */
 export default class Backend {
 
-  constructor(ipcRenderer) {
+  constructor(ipcRenderer, process) {
```

Comment:
Is there a reason we don't extract the platform using basic browser OS detection?